dev-libs/hidapi/hidapi-20160919.recipe

```diff
@@ -0,0 +1,80 @@
+SUMMARY="Library for Interfacing with HID-Class devices"
+DESCRIPTION="HIDAPI is a library working with multiple platforms for
+ interfacing with HID-Class devices through USB and Bluetooth. (On Haiku, it only works with USB). "
+HOMEPAGE="http://www.signal11.us/oss/hidapi/"
+COPYRIGHT="2010 Alan Ott
+         2011 Signal 11 Software"
+LICENSE="BSD (3-clause)
+          GNU GPL v3"
+REVISION="1"
+srcGitRev="a6a622ffb680c55da0de787ff93b80280498330f"
+SOURCE_URI="https://github.com/signal11/hidapi/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="d94e39be127ab6ac6a2bbdf6d321d6492b1cdc6cc5c00b6901228eb6923e81a6"
+SOURCE_DIR="hidapi-$srcGitRev"
+PATCHES="hidapi-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	hidapi$secondaryArchSuffix = $portVersion
+	lib:libhidapi$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libusb_1.0$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	hidapi${secondaryArchSuffix}_devel = $portVersion
+	devel:libhidapi$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_devel="
+	hidapi$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libusb_1.0$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:automake
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+
+	./bootstrap
+
+	runConfigure ./configure \
+		--disable-dependency-tracking \
+		--with-gnu-ld
+
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
```

Comment:
Please insert this:
```

	rm $libDir/libhidapi.la

```
between the "make install" and the "prepareInstalledDevelLib libhidapi".
(Yes, it's nice to have one blank line above and another below.)